pages/about.html

```diff
@@ -44,26 +44,31 @@
         <section class="home-section about-section">
             <div class="home-section-desktop-container">
```

Comment:
- [ ] [OPTIONAL] Kindly make sure the Mobile menu has a proper background that respects the accessibility rules

![Screenshot_20230601_100211](https://github.com/habitoz/serategna/assets/25227022/fb8efa8b-b4c7-43ce-928e-ad8de635f2c4)